internal/outputs/scc_collector_test.go

```diff
@@ -63,11 +62,8 @@ func TestNewSccCollector(t *testing.T) {
 	if sccCol.createSource != createSource {
 		t.Errorf("collector createSource = %v; want %v", sccCol.createSource, createSource)
 	}
-	if defaultNoThreads < 2 {
-		t.Errorf("defaultNoThreads = %v; want at least 2", defaultNoThreads)
-	}
-	if sccCol.threadsNo != defaultNoThreads {
-		t.Errorf("collector threadsNo = %v; want %v", sccCol.threadsNo, defaultNoThreads)
+	if sccCol.maxGoRoutines != defaultMaxCoroutines {
+		t.Errorf("collector threadsNo = %v; want %v", sccCol.maxGoRoutines, defaultMaxCoroutines)
```

Comment:
maybe goroutinesNo? (instead of threadsNo)

Reply:
Good idea, renaming it :-)